lib/hash_utils.rb

```diff
@@ -0,0 +1,19 @@
+# based on https://github.com/rails/rails/blob/master/activesupport/lib/active_support/core_ext/hash/keys.rb#L116
+# I chose to copy this method instead of adding activesupport as a dependency
+# because we want to have the least number of dependencies
+module FormatParser
+  class HashUtils
```

Comment:
Do we need to create an extra module just for this one method? I'm fine with attaching it to the root module. Not critical though :-)

Reply:
I understood you are suggesting to add the method `deep_transform_keys` directly in `lib/format_parser.rb`, is it? 🤔 

If it is, I prefer using an extra module and keeping it more in a "single responsibility" way.

wdyt?

Reply:
Up to you 👍